tests/e2e/const.go

```diff
@@ -46,16 +46,17 @@ const (
 	zookeeperClusterName       = "zookeeper-server"
 	managedByHelmLabelTemplate = "app.kubernetes.io/managed-by=Helm,app.kubernetes.io/instance=%s"
 
-	cruiseControlPodReadinessTimeout       = 50 * time.Second
-	kafkaClusterResourceReadinessTimeout   = 60 * time.Second
-	defaultDeletionTimeout                 = 20 * time.Second
-	defaultPodReadinessWaitTime            = 10 * time.Second
-	defaultTopicCreationWaitTime           = 10 * time.Second
-	defaultUserCreationWaitTime            = 10 * time.Second
-	kafkaClusterCreateTimeout              = 600 * time.Second
+	cruiseControlPodReadinessTimeout     = 50 * time.Second
+	kafkaClusterResourceReadinessTimeout = 60 * time.Second
+	defaultDeletionTimeout               = 20 * time.Second
+	defaultPodReadinessWaitTime          = 10 * time.Second
+	defaultTopicCreationWaitTime         = 10 * time.Second
+	defaultUserCreationWaitTime          = 10 * time.Second
+
+	kafkaClusterCreateTimeout              = 600 * time.Second // Increased from 600 to 700 for multiple kind
 	kafkaClusterResourceCleanupTimeout     = 120 * time.Second
 	kcatDeleetionTimeout                   = 40 * time.Second
-	zookeeperClusterCreateTimeout          = 4 * time.Minute
+	zookeeperClusterCreateTimeout          = 7 * time.Minute // Increased from 4 to 7 for multiple kind
```

Comment:
Optional: Not sure if we need this comment here, otherwise we could add one for each line explaining why we decided on that exact amount.

Reply:
I added these for testing purposes. Because the 2 kind cluster failed because zookeeper cluster. The more time did not help.

Reply:
I think it is ok to leave it now to know what is the original value for 1 kind cluster